addOns/webuipoc/src/main/pocs/reactWebUI/.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint and Check Formatting
```

Comment:
Remove this file and apply this:
```diff
diff --git a/addOns/webuipoc/webuipoc.gradle.kts b/addOns/webuipoc/webuipoc.gradle.kts
index d0fbcc96a2..3798e04cd4 100644
--- a/addOns/webuipoc/webuipoc.gradle.kts
+++ b/addOns/webuipoc/webuipoc.gradle.kts
@@ -49,8 +49,15 @@ for (dir in pocsSrcDir.listFiles()!!) {
                     workingDir = dir
                     args.set(arrayListOf("run", "lint"))
                 }
+            val checkFormatTask =
+                tasks.register<NpmTask>("checkFormatPoc$normalizedPocName") {
+                    group = pocBuildTasksGroup
+                    dependsOn(installTask)
+                    workingDir = dir
+                    args.set(arrayListOf("run", "check-format"))
+                }
             tasks.named(LifecycleBasePlugin.CHECK_TASK_NAME) {
-                dependsOn(lintTask)
+                dependsOn(lintTask, checkFormatTask)
             }
         }
     } else {
```

Reply:
remove whole .yml file?

Reply:
Yes, this file will not work here and there's already a CI workflow.

Reply:
![image](https://github.com/user-attachments/assets/2ec9ded6-1250-4038-aa00-06a6c5e66951)

is it fine

Reply:
ci.yml deleted

Reply:
The lines with `+` are lines that need to be added and the ones with `-` are lines that need to be removed. The `+`/`-` should not be in the final changes.

Reply:
Noted, removed 
![image](https://github.com/user-attachments/assets/4fafb125-748b-4da5-b40b-a1e2428f503c)

Reply:
You need to apply the changes taking into account the lines/context of the file. (It starts at line 49)